include/boost/spirit/home/x3/nonterminal/detail/transform_attribute.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include <boost/spirit/home/x3/support/traits/transform_attribute.hpp>
 #include <boost/spirit/home/x3/support/traits/move_to.hpp>
+#include <boost/spirit/home/x3/support/traits/value_traits.hpp>
```

Comment:
accidental stray edit, remove this when adding tests

Reply:
I'm getting error when compiling test/char_class.cpp:
```bash
xeverous/spirit/test/x3/char_class.cpp:110:26: warning: illegal character encoding in string literal [-Winvalid-source-encoding]
        BOOST_TEST(test("<E9>", alpha));
                         ^~~~
```
Is that expected?

Reply:
Yes, read a header comment in `char_class.cpp`.